db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20180406211501) do
+ActiveRecord::Schema.define(version: 20180822191538) do
```

Comment:
@vandrijevik can you try running the migration on your machine and see what sort of output you get? ran migrations from scratch to get this output because my primary database had even more noise (every table now got a `COLLATE=utf8_unicode_ci` added into the `options`). And weirdly, I needed to run `rake db:migrate` twice to get both the main app migrations and the engine migrations.

Reply:
Running `rake db:migrate` for me results in a diff for `db/schema.rb` on this branch, which includes a bunch of lines that would add `COLLATE=utf8_unicode_ci"`. I am running MySQL 5.7.23 locally.
 
Running `rake db:migrate` a second time for me did not do anything additional compared to the first run. I don’t see any migrations in `vendor/engines` in this PR, though, so I am not sure which engine migrations you are referring to?